ignore_words.txt

```diff
@@ -37,3 +37,4 @@ nin
 trough
 recuse
 ro
+nam
```

Comment:
this was the worst one, we have lots of nanoAmperes (nAm) in our plot titles / code comments.